### DISCUSSION TITLE
Key assigned to notes

### DISCUSSION THREAD

**Discussion by kur00n3k0** [ppy/osu#17360] [Ideas] [answered]
I don't know if this will be a good idea, but i was thinking on increasing the dificult adding some feature to make the player hit one note with a specific keyboard key. Eq: note 1 (press z) note 2 (press x).

To make it clear: if you press "x" at note 1 or "z" at note 2 you will lose points. I hope you guys can understand what i'm talking about hehe

**Comment by peppy** [comment, marked as answer]
Is this similar to the "alternate" mod which is already implemented?

**Reply by kur00n3k0** [reply]
> Is this similar to the "alternate" mod which is already implemented?

I don't know if its alternate mod. Let me think how i can explain that. It's like on Taiko no Tatsujin where you have blue and red notes, so you can't beat a blue with the red note key, but this in osu mode. It's something like that. If it is the alternate mod, i'm so sorry i didn't know about it ( ´･ω･` )

**Reply by frenzibyte** [reply]
It is the "Alternate" mod if you have to hit notes by different keys in succession:
```
note 1 -> hit Z (X misses)
note 2 -> hit X (Z misses)
note 3 -> hit Z (X misses)
...
```

Unless you mean keys more than Z and X would be used, in which case I'm not sure how you want that implemented.

**Reply by kur00n3k0** [reply]
I mean something like

note 1 -> hit Z (X misses)
note 2 -> hit Z (X misses)
note 3 -> hit X (Z misses)
note 4 -> hit Z (X misses)
note 5 -> hit X (Z misses)
...

The alternate mod works in that way too? Each note assigned to one key

**Reply by frenzibyte** [reply]
I don't get why the two first notes require the same key, but "Alternate" is what you want, yes.

**Reply by kur00n3k0** [reply]
If that's the case thank you! And sorry for bothering you guys with something it's already implemented

**Reply by integerrr** [reply]
> I don't get why the two first notes require the same key, but "Alternate" is what you want, yes.

I think he meant it like how taiko has it blue blue red blue red red blue blue blue for example, alt mod is just straight alternating

**Reply by kur00n3k0** [reply]
> > I don't get why the two first notes require the same key, but "Alternate" is what you want, yes.
> 
> I think he meant it like how taiko has it blue blue red blue red red blue blue blue for example, alt mod is just straight alternating 

Yes! It's exactly what i mean! Thank you for explaining better than me (´∇｀)
